### ISSUE TITLE
Has anyone tried cloning and recompiling MacDown?

### ISSUE THREAD

**Issue by manngo** [MacDownApp/macdown#1315]
I have tried and failed.

**Comment by AlexanderWillner** [comment]
Yes: https://github.com/MacDownApp/macdown/issues/1215#issuecomment-738249328

**Comment by gerardsimons** [comment]
Me too, I am getting a very long error trace on `bundle exec pod install`:

<img width="1024" alt="image" src="https://user-images.githubusercontent.com/4247845/217070972-99cf04c3-4314-414f-941a-7707540bffb4.png">
